anthos-bm-gcp-bash/install_hybrid_cluster.sh

```diff
@@ -15,7 +15,34 @@
 
 set -euo pipefail
 
-printf "✅ Using Project [%s] and Zone [%s].\n\n" "$PROJECT_ID" "$ZONE"
+if [[ -z "${PROJECT_ID}" ]]; then
+  printf "🚨 Environment variable PROJECT_ID not set. Set it to the Google Cloud Project you intend to use."
```

Comment:
nit: "... **is** not set..."?

Reply:
Fixed in 1ca2d2ffbde4c295f0abbceb1d36353a51b1a4ca